.github/.jira_sync_config.yaml

```diff
@@ -0,0 +1,21 @@
+# Sync GitHub issues to Jira issues
+
+# Configuration syntax:
+# https://github.com/canonical/gh-jira-sync-bot/blob/main/README.md#client-side-configuration
+settings:
+  # Repository specific settings
+  components: # Jira components that will be added to Jira issue
+    - mongos-vm
```

Comment:
@carlcsaposs-canonical mongo-k8s?

Reply:
I don't think `mongos-vm` and `mongos-k8s` even exist as a component in our DPE project - my understanding is that `mongodb-vm` is the component used in mongos tickets ([example](https://warthogs.atlassian.net/browse/DPE-5005))

@Mehdi-Bendriss What's the right component for mongos tickets?